.env.production

```
@@ -0,0 +1,8 @@
+# Production Environment Variables
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_cHJlY2lvdXMtaGFyZS00NC5jbGVyay5hY2NvdW50cy5kZXYk
+CLERK_SECRET_KEY=sk_test_13Inigx4tdzibOzMjTPQ91i1KUaGm7eSevPvsA0xZo
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)